Dockerfile

```diff
@@ -0,0 +1,31 @@
+# ----------------------------------------------------------------------
+# First stage, compile application
+# ----------------------------------------------------------------------
+
+FROM node:14 AS builder
+
+WORKDIR /usr/src/app
+
+# Specify Mapbox Token to use
+ARG MAPBOX_TOKEN=""
+ENV MAPBOX_TOKEN=${MAPBOX_TOKEN}
+
+# copy only package for caching purposes
+COPY package*.json /usr/src/app/
```

Comment:
We can copy everything from the project root to the work dir:
`COPY . /usr/src/app`